<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearshDialog</class>
 <widget class="QDialog" name="SearshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>查找目标：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SearshText"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>方向</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbUp">
            <property name="text">
             <string>向上(&amp;U)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDown">
            <property name="text">
             <string>向下(&amp;D)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="cbCase">
      <property name="text">
       <string>区分大小写</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="btFindNext">
      <property name="text">
       <string>查询下一个(&amp;F)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
